src/models/src/common/mod.rs

```diff
@@ -110,6 +110,12 @@ pub struct SlackClientId(pub String);
 #[derive(Debug, Eq, PartialEq, Hash, Clone, Serialize, Deserialize, ValueStruct)]
 pub struct SlackClientSecret(pub String);
 
+#[derive(Debug, Eq, PartialEq, Hash, Clone, Serialize, Deserialize, ValueStruct)]
+pub struct SlackVerificationToken(pub String);
```

Comment:
Great to have types here! 👏🏻 
Nit: 
You can make them more secure with similar approach as we have for `SlackApiTokenValue` :
```
impl std::fmt::Debug for SlackApiTokenValue {
    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
        write!(f, "SlackApiTokenValue(len:{})", self.value().len())
    }
}
```

Reply:
Nice! Tried @ d9fa27d